source/assets/forms.js

```diff
@@ -5,194 +5,146 @@ const forms = [
       trigger: 'Vill boka borgerlig vigsel',
       doneMessage: 'Då har jag tagit emot er bokning. Du kan när som helst se eller redigera din bokning under fliken Ärenden.',
       questions: [
-          {
-              key: 'partnerName',
-              question: 'Vem ska du gifta dig med?',
-              input: [
-                  {
-                      type: 'text',
-                      placeholder: 'För- och efternamn',
-                  },
-              ],
-          },
-          {
-              key: 'partnerSocialNumber',
-              question: ({ answers }) => `Vilket personnummer har ${answers.partnerName.split(' ')[0]}?`,
-              input: [
-                  {
-                      type: 'number',
-                      placeholder: 'Personnummer',
-                      maxLength: 12,
-                  },
-              ],
-          },
-          {
-              key: 'partnerSameAddress',
-              question: ({ answers }) => `Är ${answers.partnerName.split(' ')[0]} folkbokförd på samma adress som du?`,
-              input: [
-                  {
-                      type: 'radio',
-                      options: [
-                          {
-                              value: 'Ja',
-                              icon: 'check',
-                          },
-                          {
-                              value: 'Nej',
-                              icon: 'close',
-                          },
-                      ],
-                  },
-              ],
-          },
-          {
-            key: 'partnerAddress',
-            question: ({ answers }) => `Vilken adress är ${answers.partnerName.split(' ')[0]} folkbokförd på?`,
-            input: [
+        {
+              id: 'partnerName',
+              name: 'Vem ska du gifta dig med?',
```

Comment:
How come you changed from question to name? message(s)/question(s) are easier to understand imo since we will be outputting messages/questions in the chat rather then names

Reply:
I agree with you that name is not the best property. I find "text" to be better suited. But since the current api response has the property name, i found it easier/quicker to change the mockdata than the code in the service. 

This property should be changed in the service though, putting a task in jira for it (y)

I find text to be better suited since its feel more natural to have as a property on a question for example:
question.text rather than question.question